src/lib/components/workflow/workflows-summary-configurable-table/workflows-summary-table-body-cell.svelte

```diff
@@ -54,13 +54,14 @@
     updateQueryParamsFromFilter($page.url, $workflowFilters, $workflowSorts);
   };
 
-  $: cell =
-    label in WORKFLOW_CELLS ? WORKFLOW_CELLS[label] : { label, path: label };
+  $: cell = label in WORKFLOW_CELLS && WORKFLOW_CELLS[label];
   $: {
     if (isPathCell(cell)) {
       cellContent = String(workflow[cell.path]) ?? '';
     } else if (isDataCell(cell)) {
       cellContent = cell.data(workflow, $timeFormat) ?? '';
+    } else {
+      cellContent = '';
```

Comment:
Do we want null here or should it be an empty string like the other cells?

<img width="389" alt="Screenshot 2023-04-20 at 10 57 11 AM" src="https://user-images.githubusercontent.com/15069288/233436077-40b4692e-ed32-4e5a-b686-7a5197739357.png">